build.sbt

```diff
@@ -231,19 +231,32 @@ lazy val nir =
     .settings(mavenPublishSettings)
     .dependsOn(util)
 
+lazy val nirparser =
```

Comment:
Also, this project should be added to `test-all` and `scala-native/scripts/release`

Reply:
Good catch on including in `test-all`. We don't have to release it yet, as it's not part of the public API at the moment.

Reply:
a6e6257